pkg/ddl/backfilling.go

```diff
@@ -339,31 +339,44 @@ func (w *backfillWorker) handleBackfillTask(d *ddlCtx, task *reorgBackfillTask,
 	return result
 }
 
+func (w *backfillWorker) sendResult(result *backfillResult) {
+	select {
+	case <-w.ctx.Done():
+	case w.resultCh <- result:
+	}
+}
+
 func (w *backfillWorker) run(d *ddlCtx, bf backfiller, job *model.Job) {
-	logutil.BgLogger().Info("backfill worker start", zap.String("category", "ddl"), zap.Stringer("worker", w))
-	var curTaskID int
+	logger := ddlLogger.With(zap.Stringer("worker", w))
```

Comment:
Could we add "jobID" to this log? Then we can associate this log with other logs with jobID.